spotbugs/src/main/java/edu/umd/cs/findbugs/sarif/Extension.java

```diff
@@ -32,16 +32,26 @@
         this.organization = organization;
     }
 
-    JSONObject toJSONObject() {
+    JsonObject toJsonObject() {
         // TODO put 'fullDescription' with both of text and markdown representations
-        JSONObject desc = null;
+        JsonObject desc = null;
         if (shortDescription != null) {
-            desc = new JSONObject().put("text", shortDescription);
+            desc = new JsonObject();
+            desc.addProperty("text", shortDescription);
         }
-        return new JSONObject().put("version", version).put("name", name)
-                .putOpt("shortDescription", desc)
-                .putOpt("informationUri", informationUri)
-                .putOpt("organization", organization);
+        JsonObject extensionJson = new JsonObject();
+        extensionJson.addProperty("version", version);
+        extensionJson.addProperty("name", name);
+        if (desc != null) {
+            extensionJson.add("shortDescription", desc);
+        }
+        if (informationUri != null) {
+            extensionJson.addProperty("informationUri", informationUri.toString());
+        }
+        if (organization != null) {
+            extensionJson.addProperty("organization", String.valueOf(organization));
```

Comment:
Is are any reason to prefer `String.valueOf( ... )` over `organization.toString()`? If so, I want to make the intention clear.